paramiko/transport.py

```diff
@@ -217,7 +221,10 @@ class Transport (threading.Thread, ClosingContextManager):
         'diffie-hellman-group-exchange-sha256': KexGexSHA256,
         'gss-group1-sha1-toWM5Slw5Ew8Mqkay+al2g==': KexGSSGroup1,
         'gss-group14-sha1-toWM5Slw5Ew8Mqkay+al2g==': KexGSSGroup14,
-        'gss-gex-sha1-toWM5Slw5Ew8Mqkay+al2g==': KexGSSGex
+        'gss-gex-sha1-toWM5Slw5Ew8Mqkay+al2g==': KexGSSGex,
+        'ecdh-sha2-nistp256': KexNistp256,
+        'ecdh-sha2-nistp384': KexNistp384,
+        'ecdh-sha2-nistp521': KexNistp521
```

Comment:
Thanks for reviewing. Good idea about the trailing comma. I'll incorporate all the review suggestions in a single go if there are more. :)